automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspacePage.scala

```diff
@@ -69,6 +69,18 @@ abstract class WorkspacePage(namespace: String, name: String)(implicit webDriver
   def goToMonitorTab(): WorkspaceMonitorPage = {
     val page = new WorkspaceMonitorPage(namespace, name)
     clickTab("Monitor", page.url)
+    await condition {
+      try {
+        val element = find(CssSelectorQuery(s"[data-test-persistence-key='$name:monitor']"))
+        if (element.isDefined) {
+          element.get.underlying.getAttribute("data-test-state") == "ready"
```

Comment:
wait for table ready. this table doesn't have a `data-test-id`, using `data-test-persistence-key` for table identification